content/200-concepts/100-components/01-prisma-schema/06-relations/420-referential-integrity.mdx

```diff
@@ -35,37 +60,63 @@ generator js {
 }
 ```
 
-Setting the value to `prisma` will remove all foreign keys previously created in the next migration when using Prisma Migrate and the referential integrity will be handled by the client. The value `foreignKeys` creates foreign keys for all relations in the next migration when using Prisma Migrate, letting the database handle the referential integrity.
+To change the type of referential integrity used, update the `referentialIntegrity` parameter in the [`datasource`](/reference/api-reference/prisma-schema-reference#datasource) block. The available options are:
+
+- `foreignKeys`: this handles referential integrity in the database using foreign keys. This is the default option when using all SQL database connectors.
+- `prisma`: this emulates referential integrity in the Prisma client. This is the default (and only available) option when using the MongoDB connector.
+
+<Admonition type="warning">
+  When using Prisma Migrate, setting the value to `prisma` will remove all
+  previously created foreign keys in the next migration. Setting the value to
+  `foreignKeys` will create foreign keys for all relations in the next
+  migration.
+</Admonition>
 
 ## Handling the referential integrity in Prisma
 
-If the integrity is handled by the Prisma Client, for now there will be reduced amount of available [referential actions](/concepts/components/prisma-schema/relations/referential-actions) available in the data model:
+In the case where referential integrity is emulated in the Prisma client using the `prisma` option for the `referentialIntegrity` parameter, there is currently a reduced number of supported [referential actions](/concepts/components/prisma-schema/relations/referential-actions) available in the data model:
+
+| Referential action | `onUpdate` | `onDelete` |
+| :----------------- | :--------- | :--------- |
+| Restrict           | no         | yes        |
+| SetNull            | no         | yes        |
+| SetDefault         | no         | no         |
+| Cascade            | no         | yes        |
+| NoAction           | no         | yes        |
+
+Currently Prisma only implements the referential actions. Foreign keys also create constraints, which make it impossible to manipulate data in a way that would violate these constraints: instead of executing the query, the database responds with an error. These constraints will not be created if you emulate referential integrity in the client, so if you set the referential action to [`NoAction`](/concepts/components/prisma-schema/relations/referential-actions#noaction) there will be no checks to prevent you from breaking the referential integrity.
```

Comment:
```suggestion
Currently Prisma only implements some referential actions (see table). Foreign keys also create constraints, which make it impossible to manipulate data in a way that would violate these constraints: instead of executing the query, the database responds with an error. These constraints will not be created if you emulate referential integrity in the client, so if you set the referential action to [`NoAction`](/concepts/components/prisma-schema/relations/referential-actions#noaction) there will be no checks to prevent you from breaking the referential integrity.
```
Maybe?